index.html

```diff
@@ -104,9 +99,9 @@ <h2 class="face">Facebook 360</h2>
           </ul>
           <button class="btn">See project</button>
         </div>
-      </section>
+      </section> -->
       <!--section 2 of the portfolio cards starts-->
-      <section class="sectiontemplate">
+      <!-- <section class="sectiontemplate">
         <img class="img1" src="img/Portfolio2.svg" alt="Availability Page" />
         <img class="imgd2" src="img/DesktopPortfolio2.svg" alt="Availability Page" />
         <div class="Section3Text">
```

Comment:
- [ ] Kindly remove all the commented code, it makes your code unclear, and you always can find it on the git tree.